src/Uno.Extensions.Core.Generators/Rules.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+
+namespace Uno.Extensions.Reactive.Generator;
+
+[SuppressMessage("ReSharper", "InconsistentNaming")]
```

Comment:
Are ReSharper naming rules part of our coding standard? If not, we should remove this SuppressMessage; if they are, we should be enforcing this via the build

Reply:
Nope, didn't saw it, removed!